.github/workflows/build-nomad-spark-distro.yaml

```diff
@@ -0,0 +1,27 @@
+name: Build and Test Nomad-Spark Distribution
+
+on: [workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Tag/Build/Test
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: '1.8'
+      - name: Cut Tag from Branch
+        run: |
+          export VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+          git tag $VERSION-nomad-0.8.6-$(date +'%m%d%Y')
```

Comment:
I see the earlier tag convention is `v2.4.0-nomad-0.8.6-20181220`

1. `mvn help:evaluate -Dexpression=project.version -q -DforceStdout` returns me `2.4.3`.
can we add the string `v` at the begining.

2. can we change the date string to `'%Y%m%d'` so that it matches the earlier convention

Reply:
can do both, will test and re-request a review!